server/sonar-server/src/main/java/org/sonar/server/permission/PermissionService.java

```diff
@@ -160,40 +163,34 @@ public void applyDefaultPermissionTemplate(final String componentKey) {
     indexProjectPermissions();
   }
 
+  /**
+   * @deprecated since 5.2 – to be deleted once Permission Template page does not rely on Ruby
+   */
+  @Deprecated
   public void applyPermissionTemplate(Map<String, Object> params) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)